### ISSUE TITLE
Jetpack Connection: VaultPress Backup card contains unlabelled content

### ISSUE THREAD

**Issue by monsieur-z** [Automattic/jetpack#35481]
In the welcome screen, the VaultPress Backup card contains data without any context for screen readers. They read stats this way: “2, 2, 1”.

Issue reported in p1HpG7-qVW-p2.

<img width="533" alt="Screenshot 2024-02-13 at 2 40 01 PM" src="https://github.com/Automattic/jetpack/assets/1620183/b12aaa7c-1f3e-491d-a9bd-54be726f6178">

<img width="314" alt="Screenshot 2024-02-13 at 2 39 52 PM" src="https://github.com/Automattic/jetpack/assets/1620183/74b62b44-9fc6-4ef2-a90e-874ae2e764c1">

--- 

- At the very least, screen readers should read "2 pages" and "2 posts" in the example above. This could be implemented with the visually hidden technique (more detail in [this article)](https://www.a11yproject.com/posts/how-to-hide-content/).
- Icons without labels (especially if they are not "standards") lack context. Many users (not just screen reader users) could benefit from adding the words "pages" and "posts" on the card.
- Let's check if this could be an issue with other cards in the section

**Comment by jeherve** [comment]
Those icons and numbers are also discussed in 1242-gh-jetpack-roadmap.
